ReCaptchaContact/Model/WebapiConfigProvider.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Magento\ReCaptchaContact\Model;
+
+use Magento\ContactGraphQl\Model\Resolver\ContactUs;
+use Magento\ReCaptchaUi\Model\IsCaptchaEnabledInterface;
+use Magento\ReCaptchaUi\Model\ValidationConfigResolverInterface;
+use Magento\ReCaptchaValidationApi\Api\Data\ValidationConfigInterface;
+use Magento\ReCaptchaWebapiApi\Api\Data\EndpointInterface;
+use Magento\ReCaptchaWebapiApi\Api\WebapiValidationConfigProviderInterface;
+
+class WebapiConfigProvider implements WebapiValidationConfigProviderInterface
+{
+    private const CAPTCHA_ID = 'contact';
+
+    public function __construct(
+        private readonly IsCaptchaEnabledInterface $isCaptchaEnabled,
+        private readonly ValidationConfigResolverInterface $configResolver
+    ) {
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getConfigFor(EndpointInterface $endpoint): ?ValidationConfigInterface
+    {
+        if ($endpoint->getServiceMethod() === 'resolve'
+            && $endpoint->getServiceClass() === ContactUs::class) {
```

Comment:
I see in other places we have a || here. are you sure it should be &&?
Example: https://github.com/magento/security-package/blob/develop/ReCaptchaCustomer/Model/WebapiConfigProvider.php#L65-L69

Reply:
@ihor-sviziev After the research, I found out that `\Magento\ReCaptchaSendFriend\Model\WebapiConfigProvider::getConfigFor` is the most in-line with Contact form, that's why it's `&&` not `||`